addons/survey/static/src/interactions/survey_print.js

```diff
@@ -0,0 +1,25 @@
+import { Interaction } from "@web/public/interaction";
+import { registry } from "@web/core/registry";
+import { resizeTextArea } from "@web/core/utils/autoresize";
+
+export class SurveyPrint extends Interaction {
+    static selector = '.o_survey_print';
+
+    dynamicContent = {
+        ".o_survey_user_results_print": { "t-on-click": this._onPrintUserResultsClick },
+    };
+
+    start() {
+        document.querySelectorAll('textarea').forEach((textarea) => {
+            resizeTextArea(textarea);
+        });
+    }
+
+    _onPrintUserResultsClick() {
```

Comment:
No need to prefix with `_`, everything is considered as private in Owl & Colibri.